src/fuse_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -110,6 +111,7 @@
             integrity:    true,
             bkout_type:   true,
             lc_phase:     "LcStDev",
+            zeroizable:   false,
```

Comment:
From a FIPS perspective, do we need to support zeroization for UDS? 

No need to block merge on this. Added an issue to document a response. 

https://github.com/chipsalliance/caliptra-ss/issues/623